migration/planning-migration-3-to-4.adoc

```diff
@@ -0,0 +1,158 @@
+
+For more information, see xref:../architecture/architecture-installation.adoc#installation-overview_architecture-installation[OpenShift Container Platform installation overview].
+
+[discrete]
+==== Upgrading your cluster
+
+In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks. In {product-title} 4.2, The cluster manages its own updates, including updates to Red Hat Enterprise Linux CoreOS (RHCOS) on cluster nodes. You can easily upgrade your cluster by using the web console or by using the `oc adm upgrade` command from the OpenShift CLI and the Operators will automatically upgrade themselves. If your {product-title} 4.2 cluster has Red Hat Enterprise Linux workers, then you will still need to run an Ansible playbook to upgrade those workers.
+
+For more information, see xref:../updating/updating-cluster-between-minor.adoc#updating-cluster-between-minor[Updating clusters].
+
+[id="migration-considerations"]
+== Migration considerations
+
+Review the changes and other considerations that might affect your migration from {product-title} 3.11 to {product-title} 4.
+
+[id="migration-preparing-storage"]
+=== Storage considerations
+
+Review the storage changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== TODO
+
+[id="migration-preparing-networking"]
+=== Networking considerations
+
+Review the networking changes to consider when migrating from {product-title} 3.11 to {product-title} 4.2.
+
+[discrete]
+==== Network isolation mode
+
+The default network isolation mode for {product-title} 3.11 was `ovs-subnet`. The default network isolation mode for {product-title} 4.2 is now NetworkPolicy. If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant` mode, it is recommended to switch to the NetworkPolicy mode for your {product-title} 4.2 cluster. If you want to maintain the `ovs-multitenant` behavior while using NetworkPolicy in {product-title} 4.2, follow the steps to xref:../networking/configuring-networkpolicy.adoc#nw-networkpolicy-multitenant-isolation_configuring-networkpolicy-plugin[configure multitenant isolation using NetworkPolicy].
+
+For more information, see xref:../networking/configuring-networkpolicy.adoc#nw-networkpolicy-about_configuring-networkpolicy-plugin[About network policy].
+
+[discrete]
+==== Encrypting traffic between hosts
+
+In {product-title} 3.11, you could use IPsec to encrypt traffic between hosts. {product-title} 4.2 does not support IPsec. It is recommended to use Red Hat OpenShift Service Mesh to enable mutual TLS between services.
+
+For more information, see xref:../service_mesh/service_mesh_arch/understanding-ossm.adoc#understanding-ossm[Understanding Red Hat OpenShift Service Mesh].
+
+[id="migration-preparing-logging"]
+=== Logging considerations
```

Comment:
@jcantrill @richm Can you please review this logging section to make sure it's accurate and that I'm not missing anything? Thanks!

Preview: https://migration-guide--ocpdocs.netlify.com/openshift-enterprise/latest/migration/planning-migration-3-to-4.html#migration-preparing-logging